archive/xar_header.ksy

```diff
@@ -0,0 +1,89 @@
+meta:
+  id: xar_header
```

Comment:
I understand that you tried to clarify that it can only parse the header, but this applies to a number of formats already in KSF and none of them use the main identifier to denote that. In reality, the vast majority of existing `.ksy` specifications are somehow incomplete or incorrect, but let's use other means to indicate the implementation status rather than the name.

Actually I hope that the repo https://github.com/kaitai-io/kaitai_struct_samples will eventually help meet this goal - if we collect enough quality sample files and their manually curated expected interpretations (in practice probably a YAML tree of expected values), we could use these to test the `.ksy` specifications and display the score as the "implementation status". Thanks to that, it would be transparent how each specification is doing, which one requires care and what missing KS features block many of the `.ksy` implementations from being better.

Reply:
I am not attached to the name, so feel free to rename.